### PR TITLE
Pantry class beyond setup

### PR DESCRIPTION
**Pull request by MichaelEWalker87**
* This adds the test file for the pantry 

* This also continues to work out the fetch  to help assign it to a variable 

## Description

* This adds to the test file in the pantry 

* This also start the fetch process to store 

## Related Issue

* We need to figure out what to do with the data and how to store it 

## Motivation and Context

* This was added to test the functionality of the pantry and also start the process to the fetch request 

## How Has This Been Tested?

* It has been tested in by the console.log() 

* It has also been tested with the npm test  

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/pantry.js [hunk 1]
```diff
@@ -44,9 +44,9 @@ determineCost(user, recipe) {
   return estimatedCostNeeded.join('; also ')
 }
 
-addNecessay(user, recipe){
-  checkIngredients(user, recipe)
-}
+// addNecessay(user, recipe){
```
Comment (Estaffieri) [inline src/pantry.js:47]: Is this a spelling error?

**Review by Estaffieri — APPROVED** [review]
I think using a promise.all and bringing all the fetch requests into the same timelines is the best move for this kind of application.

File: src/pantry.js (hunk 1 above)
Comment (BrigetteDoelp) [reply src/pantry.js:47]: lol well at least it's a consistent spelling error? good catch though!

**Comment by BrigetteDoelp** [comment]
Save for the spelling error, this looks great! I've made a note of it so we can fix it on the next branch. Let's merge!

**Pull request merged**